src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -940,7 +940,7 @@ public void onPlayerDeath(PlayerDeathEvent event) {
 			if (BENDING_PLAYER_DEATH.containsKey(event.getEntity())) {
 				String message = ConfigManager.languageConfig.get().getString("DeathMessages.Default");
 				String ability = BENDING_PLAYER_DEATH.get(event.getEntity());
-				String tempAbility = ChatColor.stripColor(ability).replaceAll(" ", "");

```

Comment:
I am ok with this change, however, I think it would be good to add a console warning informing the server that there are potential side effects of having an Ability with spaces in the name. Something along the lines of

`Warning: <Ability Name> contains a delimiter in its name and won't work properly.`